entry/src/test/java/ledongli/safepreference/ExampleUnitTest.java

```diff
@@ -2,16 +2,14 @@
 
 import org.junit.Test;
 
-import me.dozen.sample.Test;
-
 import static org.junit.Assert.*;
 
 /**
  * To work on unit tests, switch the Test Artifact in the Build Variants view.
  */
 public class ExampleUnitTest {
     @Test
-    public void addition_isCorrect() throws Exception {
+    public void additionisCorrect() {
```

Comment:
Test case not relevant to library. Please remove

Reply:
Done